chaoslib/litmus/pod_delete/kill_random_pod.yml

```diff
@@ -107,13 +97,29 @@
         retries: 90   
       when: "type is defined and type != ''"
 
-    when: "chaosDiffTimeStamp is not defined or (chaosDiffTimeStamp is defined and chaosDiffTimeStamp|int < c_duration|int)"
+    - name: "[Status]: Verify that the pod delete timestamp pod is running"
```

Comment:
Let us follow the same approach for termination as used in network-chaos and other experiments